tftrt/examples/image-classification/image_classification.py

```diff
@@ -52,29 +52,34 @@ def after_run(self, run_context, run_values):
                 current_step, self.num_steps, duration * 1000,
                 self.batch_size / self.iter_times[-1]))
 
-class DurationHook(tf.train.SessionRunHook):
+class BenchmarkHook(tf.train.SessionRunHook):
```

Comment:
Could you update the docs here explaining what `target_duration` and `iteration_limit` are exactly.